core/extensions.rs

```diff
@@ -521,6 +542,27 @@ impl ExtensionBuilder {
     self
   }
 
+  /// Consume the [`ExtensionBuilder`] and return an [`Extension`].
+  pub fn take(self) -> Extension {
```

Comment:
What's the difference between `build` and `take`? Could we get rid of one of them in a follow up?

Reply:
`build` is currently used by some code using the fluent extension builder pattern, but we can definitely take it out in a future PR.